SPECS/mlnx-ofa_kernel-hwe/mlnx-ofa_kernel-hwe.spec

```diff
@@ -139,7 +139,7 @@ BuildRequires: /usr/bin/perl
 %description
 InfiniBand "verbs", Access Layer  and ULPs.
 Utilities rpm.
-The driver sources are located at: http://www.mellanox.com/downloads/ofed/mlnx-ofa_kernel-24.10-0.7.0.tgz
+The driver sources are located at: http://www.mellanox.com/downloads/ofed/mlnx-ofa_kernel-25.07-0.9.7.tgz
```

Comment:
Suggestions: you could replace hard-coded version with the macros from the top of the file, so we don't have to update them separately.